runtime/src/protocol.rs

```diff
@@ -140,6 +140,11 @@ impl Protocol {
         &self.config
     }
 
+    /// The runtime attestation key.
+    pub fn get_rak(&self) -> &RAK {
```

Comment:
Does this ever need the private component, or can this just return the public key?

Reply:
Can return just the public key. Good idea, will update.